src/App.jsx

```diff
@@ -14,70 +15,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
+  const clearSelectedList = () => setSelectedGood('');
+  const getSelectedClassName = good => (
```

Comment:
`getSelectedClassName`, `clearSelectedList` and declaration state
It’s better to separate them by lines, it will be easier to read